See: http://gcc.gnu.org/bugzilla/show_bug.cgi?id=37220

2008-06-28  Andrew Jenner  <andrew@codesourcery.com>

	* regrename.c (build_def_use): Don't copy RTX.

---
 gcc/regrename.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: gcc-4.3.2-RC-20080819/gcc/regrename.c
===================================================================
--- gcc-4.3.2-RC-20080819.orig/gcc/regrename.c
+++ gcc-4.3.2-RC-20080819/gcc/regrename.c
@@ -813,7 +813,7 @@ build_def_use (basic_block bb)
 		    OP_IN, 0);
 
 	  for (i = 0; i < recog_data.n_dups; i++)
-	    *recog_data.dup_loc[i] = copy_rtx (old_dups[i]);
+	    *recog_data.dup_loc[i] = old_dups[i];
 	  for (i = 0; i < n_ops; i++)
 	    *recog_data.operand_loc[i] = old_operands[i];
 	  if (recog_data.n_dups)
